pkg/kudoctl/cmd/root.go

```diff
@@ -74,9 +73,5 @@ and serves as an API aggregation layer.
 func initGlobalFlags(cmd *cobra.Command, out io.Writer) {
 	flags := cmd.PersistentFlags()
 	Settings.AddFlags(flags)
-	clog.Init(flags, out)
-	// FIXME: add error handling
-	cmd.ParseFlags(os.Args[1:])
```

Comment:
This `ParseFlags()` call had no effect. Env var overrides are now taken directly as defaults when adding the cobra flag.